tools/go-cli/build.gradle

```diff
@@ -0,0 +1,75 @@
+ext.dockerImageName = "whisk/go-cli-test"
+ext.dockerContainerName = "go-cli-test"
+apply from: "../../docker.gradle"
+
+ant.condition(property: "family", value: "linux") {
+    os(family: "unix")
+}
+
+ant.condition(property: "family", value: "mac") {
+    os(family: "mac")
+}
+
+ant.condition(property: "arch", value: "amd64") {
+    os(arch: "amd64")
+}
+
+ant.condition(property: "arch", value: "amd64") {
+    os(arch: "x86_64")
+}
+
+ant.condition(property: "arch", value: "386") {
+    os(arch: "386")
+}
+
+ant.condition(property: "arch", value: "386") {
+    os(arch: "i386")
+}
+
+ant.condition(property: "arch", value: "386") {
+    os(arch: "x86_32")
+}
+
+task removeBinary(type: Delete) {
+    delete "${projectDir}/../../bin/go-cli"
+    delete "${projectDir}/wsk"
+}
+
+task distBinary(dependsOn: [removeBinary, distDocker]) << {
+
+    run(dockerBinary + ["rm", "-f", dockerContainerName], true)
+
+    run(["rm", "-rf", "${projectDir}/../../bin/go-cli/"], true)
```

Comment:
We could create a symlink in the openwhisk/bin folder like there is for the Python CLI. It appears that the only reason the Python CLI does this is because Python doesn't generate binaries. I'm not sure that the extra indirection is needed.